source/_integrations/linkplay.md

```diff
@@ -0,0 +1,21 @@
+---
+title: LinkPlay
+description: Connect and control your LinkPlay media players using the LinkPlay integration
+ha_category:
+  - Media player
+ha_domain: linkplay
+ha_zeroconf: true
+ha_integration_type: integration
+ha_release: 2024.8
+ha_codeowners:
+  - '@Velleman'
```

Comment:
```suggestion
  - '@Velleman'
```
Is Velleman here listed as code owner on purpose? Instead of you? It seems not only the docs but also the core commits were done by you.

Reply:
Velleman is the company I work for and they are effectively code owner, even though I have made the commits